.github/workflows/docker-build-release.yml

```diff
@@ -20,3 +20,4 @@ jobs:
       uploadImageAsTarball: docker-build-release
       parca: true
       buildIndividually: true
+      pushToDockerHub: true # won't actually push, just creates the tags
```

Comment:
Should we rename this parameter if it no longer means that we are pushing to docker hub?

Reply:
it would mean push if we didnt have upload image as tarball set